phpdotnet/phd/Reader.php

```diff
@@ -24,7 +27,7 @@ public function readContent(?string $node = null): string {
             if (\LIBXML_VERSION >= 20620) {
                 return self::readString();
             }
-            v("You are using libxml2 v%d, but v20620 or newer is preferred", \LIBXML_VERSION, VERBOSE_OLD_LIBXML);
+            $this->outputHandler->v("You are using libxml2 v%d, but v20620 or newer is preferred", \LIBXML_VERSION, VERBOSE_OLD_LIBXML);
 
             $node = $this->name;
```

Comment:
Aside, how old/recent is this version? Might make sense to drop this in a follow-up.

Reply:
It's from July of 2005. :-)